.github/workflows/common_actions.yml

```diff
@@ -61,14 +63,18 @@ jobs:
         working-directory: web
         run: |
             npm version patch
+            git config --local user.name 'github-actions[bot]'
+            git config --local user.email 'github-actions[bot]@users.noreply.github.com'
             git add .
             git commit -m "Update package.json"
-        if: github.event_name == 'push' && github.ref_name == 'web'
+        if: github.event_name == 'push' && github.ref_name == 'api'
```

Comment:
Ricordati che d'ora in poi se vuoi i changes del package.json allora bisogna chiamare sempre la branch api

Reply:
yep